types/share/share.go

```diff
@@ -2,9 +2,30 @@ package share
 
 import (
 	"github.com/celestiaorg/nmt"
+	"github.com/celestiaorg/rsmt2d"
+
 	"github.com/rollkit/celestia-openrpc/types/core"
 )
 
+var (
+	// DefaultRSMT2DCodec is the default codec creator used for data erasure.
+	DefaultRSMT2DCodec = rsmt2d.NewLeoRSCodec
+)
+
+const (
+	// NamespaveVersionSize is the size of a namespace version in bytes.
+	NamespaceVersionSize = 1
+
+	// NamespaceIDSize is the size of a namespace ID in bytes.
+	NamespaceIDSize = 28
+
+	// NamespaceSize is the size of a namespace (version + ID) in bytes.
+	NamespaceSize = NamespaceVersionSize + NamespaceIDSize
+
+	// ShareSize is the size of a share in bytes.
+	ShareSize = 512
```

Comment:
let's also bring other constants related to shares: https://github.com/celestiaorg/celestia-app/blob/main/pkg/appconsts/global_consts.go#L39, so that you can use them in rollkit later on. For example, you could use `ShareVersionZero` in calling `NewBlob(`.

Reply:
Created a new package `appconsts` as these consts were being used across multiple packages.